### PR TITLE
Change the upper limit of required version from 7.2.x to 7.3.x.

### PR DESCRIPTION
**Pull request by kunit**
Changed package.xml and added .travis.yml.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  apt:
+    packages:
+      - libmcrypt-dev
+    update: true
+
+before_script:
+  - phpize && ./configure && make
+
+script:
+  - make test REPORT_EXIT_STATUS=1 NO_INTERACTION=1 TESTS="--show-all" | tee tests-output.txt && if grep -q 'TEST SUMMARY$' tests-output.txt ; then exit 1 ; fi
```
Comment (nikic) [inline .travis.yml]: What's the reason for the tests-output.txt and grep here? Should just the make test run be sufficient? It seems to pass on Travis (which I just enabled).

File: package.xml [hunk 2]
```diff
@@ -131,6 +131,16 @@
...
+        <release>
+            <date>2018-12-23</date>
+            <version>
+                <release>1.0.2</release>
+                <api>1.0.0</api>
+            </version>
+            <notes>
+                - Changed maximum version to 7.3.x.
+            </notes>
+        </release>
```
Comment (nikic) [inline package.xml]: I believe this change is only made after the release.

File: .travis.yml (hunk 1 above)
Comment (kunit) [reply .travis.yml]: There was no need to grep tests-output.txt.
I fixed it so as not to grep and committed it.

File: package.xml (hunk 2 above)
Comment (kunit) [reply package.xml]: I reverted package.xml.

**Comment by kunit** [comment]
@nikic Please review again.

**Comment by nikic** [comment]
Looks fine to me.

@remicollet Are you able to publish new versions for this package?

**Comment by remicollet** [comment]
@nikic sorry, no, I don't have access to this one

Probably @derickr can help here

(an least giving one of us, proper pecl permission)

**Comment by derickr** [comment]
I'm happy to sort this out when I get back on the 4th. Please ping me then in case I forget

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+configure.ac
+install-sh
+libtool
+ltmain.sh
+missing
+mkinstalldirs
+modules
+run-tests.php
+
+tests-output.txt
```
Comment (tenkoma) [inline .gitignore]: Does this line will not became unnecessary?

File: php_mcrypt.h [hunk 4]
```diff
@@ -29,7 +29,7 @@
 extern zend_module_entry mcrypt_module_entry;
 #define mcrypt_module_ptr &mcrypt_module_entry
 
-#define PHP_MCRYPT_VERSION "1.0.1"
+#define PHP_MCRYPT_VERSION "1.0.2"
```
Comment (tenkoma) [inline php_mcrypt.h]: Is it also necessary to revert here?

**Comment by kunit** [comment]
@tenkoma thanks!

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+*.o
+
+*.swp
+.*.rej
+*.rej
+.*~
+*~
+.#*
+.DS_Store
+Thumbs.db
```
Comment (derickr) [inline .gitignore]: These last two should really be in your global .gitignore.

File: php_mcrypt.h [hunk 6]
```diff
@@ -2,7 +2,7 @@
    +----------------------------------------------------------------------+
    | PHP Version 7                                                        |
    +----------------------------------------------------------------------+
-   | Copyright (c) 1997-2017 The PHP Group                                |
+   | Copyright (c) 1997-2018 The PHP Group                                |
```
Comment (derickr) [inline php_mcrypt.h]: This file hasn't changed, so please don't update the year.

**Review by derickr — CHANGES_REQUESTED** [review]
I have some questions, which I think should be resolved. Also please squash the commits once the issues are resolved.

File: .gitignore (hunk 5 above)
Comment (kunit) [reply .gitignore]: Removed `.DS_Store` and `Thumbs.db` from .gitignore.

File: php_mcrypt.h (hunk 6 above)
Comment (kunit) [reply php_mcrypt.h]: Reverted php_mcrypt.h

**Comment by kunit** [comment]
@derickr fixed `.gitignore` and `php_mcrypt.h`. Please review again.

**Pull request merged**